packages/containers/src/DeleteResource/DeleteResource.container.js

```diff
@@ -70,15 +81,25 @@ export default class DeleteResource extends React.Component {
 		const resourceInfo = this.getResourceInfo();
 		const validateAction = this.getActions(deleteResourceConst.VALIDATE_ACTION, resourceInfo);
 		const cancelAction = this.getActions(deleteResourceConst.CANCEL_ACTION, resourceInfo);
+		const i18nKey = this.props.female
+			? 'DELETE_RESOURCE_MESSAGE_female'
+			: 'DELETE_RESOURCE_MESSAGE';
 		return (
 			<ConfirmDialog
 				show
 				header={this.props.header}
 				cancelAction={cancelAction}
 				validateAction={validateAction}
 			>
-				<div>{resourceInfo.label}</div>
+				<div>
+					<Trans i18nKey={i18nKey}>
```

Comment:
2 things on i18n : 
* add npm scripts to extract translation keys in root package.js
* this trans is not compatible with i18next-parser, I won't block this PR for that since it is really needed and the solution is the good one I think, but we need a solution

Reply:
1) The npm script is on the container package.json
2) Currently yes, there is some work here on this side -> https://github.com/i18next/i18next-parser/pull/85

For now, i think this is still the best way to get the translations with html markup.
There is another issue currently, as the Trans component manage to make it work the count attribute but the "context" attribute to declare male or female  translation is not working as it is. I try to make a PR on this point in react-i18next ;)

Reply:
1. it should be added to the root `package.json` like the others, so that `yarn extract-i18n` extract all packages translations in a single folder. Take a look at root npm scripts